build.sbt

```diff
@@ -40,11 +40,9 @@ lazy val publishSettings = Seq(
 val awsSdkVersion = "1.11.119"
 val circeVersion  = "0.7.1"
 val slf4jVersion  = "1.7.25"
-val scala212      = "2.12.2"
-val scala210      = "2.10.6"
 
 lazy val root = (project in file("."))
-  .enablePlugins(CrossPerProjectPlugin)
+  .enablePlugins()
```

Comment:
And so this is redundant?

Reply:
Good point. I was so busy removing the things that made the compiler unhappy that I didn't think about whether what was left was needed.